src/main/java/org/gridsuite/study/server/repository/SensitivityParametersEntity.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.repository;
```

Comment:
You should create a sensitivity package in repository to group all the sensi entities